packages/telegram-stamper/src/index.ts

```diff
@@ -0,0 +1,155 @@
+/// <reference lib="dom" />
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { TelegramStamperError } from "./errors";
+
+declare global {
+  interface Window {
+    Telegram: any;
+  }
+}
+
+// config when telegram stamper is passed in an api key
+export type TTelegramStamperConfig = {
+  apiPublicKey: string;
+  apiPrivateKey: string;
+};
+
+const PUBLIC_KEY_CLOUD_STORAGE_KEY = "TURNKEY_API_PUBLIC_KEY";
+const PRIVATE_KEY_CLOUD_STORAGE_KEY = "TURNKEY_API_PRIVATE_KEY";
```

Comment:
Pull from env vars?

Reply:
I wanted to keep these constant from application to application was the reason I didn't make these configurable, having said that, im not really sure what the best approach here is

Reply:
These are now semi-configurable through the cloudStorageKeySuffix optional parameter in the constructor